lib/quake_parser.ex

```diff
@@ -22,17 +39,17 @@ defmodule QuakeParser do
   defp find_games(log_content) do
     [_h | games] = String.split(log_content, "InitGame:")
 
-    games |> Enum.map(fn str -> String.split(str, "\n") end)
+    games |> Enum.map(&String.split(&1, "\n"))
```

Comment:
```suggestion
    Enum.map(games, &String.split(&1, "\n"))
```

Credo recomenda pipe somente com mais de 3 operações